algorithms/classification/regression/preprocess.py

```diff
@@ -5,7 +5,7 @@
 from sklearn.feature_selection import RFE
 
 def preprocess_train_data():
-    train_data = loaddata.load_train_data("./data/titanic-dataset","train.csv")
+    train_data = loaddata.load_train_data("../dataset/titanic","train.csv")
```

Comment:
This statement is repeated thrice in the file and requires you to enter file path again, how about moving this into a common method, say `get_data` ?

Reply:
good point. noted .

Reply:
can be also done using a constant for the base path for dataset folder  and specific path for individual dataset can be provided as a parameter to the function and it can return the complete path .

Reply:
That's cherry on the cake 😉 I like it.